webpack.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.jsx?$/,
```

Comment:
should this test js files as well? I think it should also exclude `test.jsx` files.